operator/src/main/java/oracle/kubernetes/operator/logging/ThreadLoggingContext.java

```diff
@@ -27,6 +28,10 @@ public ThreadLoggingContext presenceInfo(DomainPresenceInfo info) {
     return namespace(info.getNamespace()).domainUid(info.getDomainUid());
   }
 
+  public ThreadLoggingContext presenceInfo(ClusterPresenceInfo info) {
+    return namespace(info.getNamespace());
```

Comment:
Will this be confusing when there are multiple clusters (potential with same name) in the same namespace?

Reply:
I don't think we can have multiple same-named clusters in the same namespace; K8S disallows it.

Reply:
ok. As we discussed, I meant the name of the cluster itself and not the name of the cluster resource. Should we also include the cluster resource name is the ThreadLoggingContext?

Reply:
We probably should if we also want the cluster resource name to be in the logging headers.